src/clj_http/conn_mgr.clj

```diff
@@ -232,20 +247,26 @@
       get-custom-strategy-registry))
 
 (defn ^BasicHttpClientConnectionManager make-regular-conn-manager
-  [{:keys [keystore trust-store
+  [{:keys [custom-dns-resolver
```

Comment:
To match the consistency of other custom options, prefer naming this `dns-resolver`.